packages/react-google-maps-api/src/LoadScriptNext.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+
+import { LoadScriptProps, DefaultLoadingElement } from './LoadScript'
+import { useLoadScript } from './useLoadScript'
+
+export interface LoadScriptNextProps extends LoadScriptProps {
+  loadingElement: React.ReactElement
+  children: React.ReactElement
+}
+
+export const LoadScriptNext = React.memo<LoadScriptNextProps>(
```

Comment:
Please declare a component before wrapping it to React.memo. performance reasons.

Reply:
Huh? Never head that one before. How is more performant if it's separated?

Reply:
The correct way to declare a component as function first:
```jsx
function MyComponent () {
  return <JSX />
}
```

then you export it wrapped in memo:
```jsx
export default React.memo(MyComponent)
```

otherwise it ruins React debug with <undefined /> components.

Reply:
React does check for Component.displayName, and React.memo does not provide it for us.

Reply:
So you mean "debugging reasons". You got me fairly confused how that can be related to performance 😆

Reply:
sorry)